h/static/extension/config.js

```diff
@@ -1,8 +0,0 @@
-function hypothesisConfig() {
-  // Pages on our site can include a meta tag to trigger specific behaviour
-  // when the extension loads.
-  var hypothesisIntent = document.querySelector('[name="hypothesis-intent"]');
```

Comment:
Sorry if this is a dumb question, but if we've removed this, then how do we now trigger the extension to open automatically on the welcome page after installation?